src/timesignature.ts

```diff
@@ -18,18 +18,18 @@ export interface TimeSignatureInfo {
 }
 
 const assertIsValidFraction = (timeSpec: string) => {
```

Comment:
Maybe `assertIsValidTimeSig` since the implementation is different now

Reply:
Done